libsplinter/src/peer/connector.rs

```diff
@@ -263,10 +263,10 @@ pub(crate) struct PeerRemover {
 impl PeerRemover {
     /// This function will only be called when the PeerRef is dropped.
     ///
-    /// Sends a request to the PeerManager to remove a peer.
+    /// Sends a request to the `PeerManager` to remove a peer.
     ///
     /// # Arguments
-    /// * `peer_id` - the peer_id of the PeerRef that has been dropped
+    /// * `peer_id` - the peer_id of the `PeerRef` that has been dropped
```

Comment:
Inconsistent. Other descriptions say "the peer id" (that is, no underscore in the term that occurs after the dash).